### DISCUSSION TITLE
Challenges in Configuring Recurring Payments for Varied Monthly Schedules

### DISCUSSION THREAD

**Discussion by jonathanathe** [firefly-iii/firefly-iii#8353] [Firefly III] [answered]
Hello everyone, I wonder if you could help me. I'm trying to create two recurring transactions.

One of them, a payment on the 1st of the months: January, April, July, and October, had no issues. I just had to set the payment for the 1st of each month with a gap of 2.

The problem arises when I want to create the payment for the rest of the months: February, March, May, June, August, September, November, and December. How can I set up this recurring transaction, since I can't make it two consecutive payments with a gap due to how it works?

Thanks in advance!

**Comment by BackSeat** [comment, marked as answer]
Could you set up two more recurring payments, Feb with a gap of 2 and March with a gap of 2?

On Sat, 6 Jan 2024, at 18:57, jonathanathe wrote:
> 
> 
> Hello everyone, I wonder if you could help me. I'm trying to create two recurring transactions.
> 
> One of them, a payment on the 1st of the months: January, April, July, and October, had no issues. I just had to set the payment for the 1st of each month with a gap of 2.
> 
> The problem arises when I want to create the payment for the rest of the months: February, March, May, June, August, September, November, and December. How can I set up this recurring transaction, since I can't make it two consecutive payments with a gap due to how it works?
> 
> Thanks in advance!
> 
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/orgs/firefly-iii/discussions/8353>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAGJ6YKMTZXHB2Q2VLYKJODYNGNCJAVCNFSM6AAAAABBPXROFWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGAZTKNRRGE>.
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
> 

-- 
"You can lead a horse to water but you can't make it learn grammar" - Reddit

**Reply by jonathanathe** [reply]
Thank you for your response. If there's no other option, I'll do as you suggest, but it would be great to be able to open a calendar for months and directly select the months for the transaction. Likewise, thank you very much for your response, and happy New Year.
